components/vf-collapse/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-### 0.1.1
+### 0.0.1
 
 * Deprecated `vf-collapse` component. `vf-details` component needs to be used instead. [Tracking issue](https://github.com/visual-framework/vf-core/issues/1911)
+* Changelog correction [Tracking issue](https://github.com/visual-framework/vf-core/issues/2035)
 
-### 0.1.0
+### 0.0.2
```

Comment:
Version order is different then other components

Reply:
Fixed the order.